### PR TITLE
lint cleanup of influxql

### PR DESCRIPTION
**Pull request by cbushko**
In reference to [Enable `golint` on the code base](https://github.com/influxdata/influxdb/issues/4098)

- [x] Rebased/mergable
- [x] Tests pass
- [x] Sign [CLA](https://influxdata.com/community/cla/) (if not already signed)

### CODE REVIEW

File: coordinator/statement_executor.go [hunk 1]
```diff
@@ -223,10 +223,8 @@ func (e *StatementExecutor) executeAlterRetentionPolicyStatement(stmt *influxql.
 	}
 
 	// Update the retention policy.
-	if err := e.MetaClient.UpdateRetentionPolicy(stmt.Database, stmt.Name, rpu, stmt.Default); err != nil {
-		return err
-	}
-	return nil
+	err := e.MetaClient.UpdateRetentionPolicy(stmt.Database, stmt.Name, rpu, stmt.Default)
+	return err
```
Comment (e-dard) [inline coordinator/statement_executor.go]: `return e.MetaClient.UpdateRetentionPolicy(stmt.Database, stmt.Name, rpu, stmt.Default)`

File: coordinator/statement_executor.go [hunk 2]
```diff
@@ -1103,15 +1101,12 @@ func (e *StatementExecutor) writeInto(w pointsWriter, stmt *influxql.SelectState
 		return err
 	}
 
-	if err := w.WritePointsInto(&IntoWriteRequest{
+	werr := w.WritePointsInto(&IntoWriteRequest{
```
Comment (e-dard) [inline coordinator/statement_executor.go]: `return w.WritePointsInto(&IntoWriteRequest{`

File: internal/meta_client.go [hunk 3]
```diff
@@ -51,120 +51,158 @@ type MetaClientMock struct {
 	UsersFn                  func() []meta.UserInfo
 }
 
+// Close a MetaClientMock
 func (c *MetaClientMock) Close() error {
 	return c.CloseFn()
 }
 
+// CreateContinuousQuery creates a mock of MetaClient.CreateContinuousQuery
```
Comment (e-dard) [inline internal/meta_client.go]: These methods don't really "create mocks". What they do is call the `xxxFn` methods on the Mock type. I would change all of them to say the following:

```
CreateContinuousQuery calls the CreateContinuousQueryFn method on the mock MetaClient.
```

Obviously for each method you would switch out `CreateContinuousQuery` for the method's name.

File: internal/tsdb_store.go [hunk 4]
```diff
@@ -46,94 +46,155 @@ type TSDBStoreMock struct {
...
 
+// BackupShard creates a mock of TSDBStore.BackupShard
 func (s *TSDBStoreMock) BackupShard(id uint64, since time.Time, w io.Writer) error {
 	return s.BackupShardFn(id, since, w)
 }
+
+// Close creates a mock of TSDBStore.Close
 func (s *TSDBStoreMock) Close() error { return s.CloseFn() }
+
+// CreateShard creates a mock of TSDBStore.CreateShard
```
Comment (e-dard) [inline internal/tsdb_store.go]: The same rules apply to these methods as the ones on the `MetaClientMock`

**Comment by jsternberg** [comment]
I'm going to close this since it's outdated and we haven't seen any response in awhile.

**Pull request closed without merging**